go.mod

```diff
@@ -14,6 +14,7 @@ require (
 )
 
 require (
+	github.com/AlekSi/pointer v1.2.0 // indirect
```

Comment:
we can get rid of this dependency, we already maintain our own via `convert_types.go`